c/zis/client.c

```diff
@@ -138,17 +138,19 @@ static int authRequest(const CrossMemoryServerName *serverName,
 }
 
 /*
- *   safIdt - a buffer for SAF IDT token. The buffer must be ZIS_AUTH_SERVICE_PARMLIST_SAFIDT_LENGTH + 1 bytes long.
+ *   safIdt - a buffer for SAF IDT token. The buffer must be ZIS_AUTH_SERVICE_SAFIDT_LENGTH + 1 bytes long.
+ *   appl - application name to be included in the audience claim of SAF IDT. It is an optional parameter and can be null.
  */
-int zisGenerateOrValidateSafIdt(const CrossMemoryServerName *serverName,
+int zisGenerateOrValidateSafIdtWithAppl(const CrossMemoryServerName *serverName,
```

Comment:
Do we really need to pass the password to generate or verify a SAF IDT value?

Reply:
password is there for safverify https://github.com/zowe/zss/blob/3b52c8fa7295e0cd81dc01608f7683fe503f48a9/c/zis/services/auth.c#L47

Reply:
So the RACF service cannot function without the user password?

Reply:
We're trying to make sure this isn't opening up any way to use that in a bad way.

Reply:
how would racf verify the user if there is no password?